db/migrate/20190116150702_remove_empty_addresses.rb

```diff
@@ -0,0 +1,27 @@
+class RemoveEmptyAddresses < ActiveRecord::Migration[5.2]
```

Comment:
I have tested this migration locally and on dev - it successfully removed all empty addresses and nullified the address id in the table referring to it